server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/custom/CustomCapabilitiesModule.java

```diff
@@ -0,0 +1,36 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.custom;
+
+import org.apache.james.jmap.core.Capability;
+import org.apache.james.jmap.core.CapabilityIdentifier;
+import org.apache.james.jmap.core.CustomCapability;
+import org.apache.james.jmap.core.CustomCapabilityProperties;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.ProvidesIntoSet;
+
+public class CustomCapabilitiesModule extends AbstractModule {
```

Comment:
This class should be in a test package

I suggest putting this in CustomMethodContract.scala